src/core_modules/capture-core/components/Pages/EnrollmentAddEvent/EnrollmentAddEventPage.epics.js

```diff
@@ -35,3 +38,20 @@ export const saveNewEventFailedEpic = (action$: InputObservable) =>
             return batchActions([saveFailed(), rollbackEnrollmentEventWithoutId(meta.uid)]);
         }),
     );
+
+export const updateEventSucceededEpic = (action$: InputObservable) =>
```

Comment:
Hey, 
Is there any reason why `updateEventSucceededEpic` and `updateEventFailedEpic` epics are inside `EnrollmentAddEventPage.epics.js`? Because the re-schedule feature is on the `EnrollmentEditEvent` page I believe they should be moved to the edit page folder. 
WDYT?